src/System.ServiceModel.Syndication/tests/Utils/XmlDiff.cs

```diff
@@ -33,18 +32,15 @@ internal enum DiffType
 
     public class XmlDiff
     {
-        private XmlDiffDocument _SourceDoc;
-        private XmlDiffDocument _TargetDoc;
+        private XmlDiffDocument _sourceDoc;
+        private XmlDiffDocument _targetDoc;
 
-        private XmlTextWriter _Writer;     // Writer to write out the result
-        private StringBuilder _Output;
-
-        // Option Flags
-        private XmlDiffOption _XmlDiffOption = XmlDiffOption.None;
+        private XmlTextWriter _writer;     // Writer to write out the result
+        private StringBuilder _output;
 
         public XmlDiff()
         {
-            _XmlDiffOption = XmlDiffOption.IgnoreEmptyElement |
+            Option = XmlDiffOption.IgnoreEmptyElement |
                              XmlDiffOption.IgnoreWhitespace |
```

Comment:
Realign subsequent lines due to "Option".Length < "_XmlDiffOption".Length

Reply:
Done